packages/dds/tree/src/test/simple-tree/object.spec.ts

```diff
@@ -41,161 +45,199 @@ export function testObjectPrototype(proxy: object, prototype: object) {
+					// eslint-disable-next-line no-prototype-builtins -- compatibility test
+					assert.equal(asObject.isPrototypeOf(Object.create(asObject)), true);
 				});
 
-				describe("properties inherited from Object.prototype", () => {
-					for (const [key, descriptor] of Object.entries(
-						Object.getOwnPropertyDescriptors(Object.prototype),
-					)) {
-						it(`Object.getOwnPropertyDescriptor(${pretty(
-							initialTree,
-						)}, ${key}) -> ${pretty(descriptor)}`, () => {
-							const root = hydrate(schema, initialTree);
-							assert.deepEqual(
-								Object.getOwnPropertyDescriptor(findObjectPrototype(root), key),
-								descriptor,
-								`Proxy must expose Object.prototype.${key}`,
-							);
-						});
-					}
+				it(`${pretty(initialTree)}.isPrototypeOf(root) -> false`, () => {
+					const root = hydrate(schema, initialTree);
+					const asObject = root as object;
+					// eslint-disable-next-line no-prototype-builtins -- compatibility test
+					assert.equal(asObject.isPrototypeOf(asObject), false);
 				});
+			});
 
-				describe("methods inherited from Object.prototype", () => {
-					it(`${pretty(initialTree)}.isPrototypeOf(Object.create(root)) -> true`, () => {
-						const root = hydrate(schema, initialTree);
-						const asObject = root as object;
-						// eslint-disable-next-line no-prototype-builtins -- compatibility test
-						assert.equal(asObject.isPrototypeOf(Object.create(asObject)), true);
-					});
+			describe(`${pretty(initialTree)}.propertyIsEnumerable`, () => {
+				for (const key of Object.getOwnPropertyNames(initialTree)) {
+					const expected = Object.prototype.propertyIsEnumerable.call(initialTree, key);
 
-					it(`${pretty(initialTree)}.isPrototypeOf(root) -> false`, () => {
+					it(`${key} -> ${expected}`, () => {
 						const root = hydrate(schema, initialTree);
 						const asObject = root as object;
 						// eslint-disable-next-line no-prototype-builtins -- compatibility test
-						assert.equal(asObject.isPrototypeOf(asObject), false);
+						assert.equal(asObject.propertyIsEnumerable(key), expected);
 					});
-				});
-
-				describe(`${pretty(initialTree)}.propertyIsEnumerable`, () => {
-					for (const key of Object.getOwnPropertyNames(initialTree)) {
-						const expected = Object.prototype.propertyIsEnumerable.call(
-							initialTree,
-							key,
-						);
-
-						it(`${key} -> ${expected}`, () => {
-							const root = hydrate(schema, initialTree);
-							const asObject = root as object;
-							// eslint-disable-next-line no-prototype-builtins -- compatibility test
-							assert.equal(asObject.propertyIsEnumerable(key), expected);
-						});
-					}
-				});
-			}
-		});
+				}
+			});
+		}
+	});
 
-		function test1(fn: (subject: object) => unknown) {
-			for (const { schema, initialTree } of testCases) {
-				const real = structuredClone(initialTree) as object;
-				const expected = fn(real);
+	function test1(fn: (subject: object) => unknown) {
```

Comment:
could you give this a descriptive name. From its signature all I can tell is its a test that does something with a function that takes any object.